.wordlist.txt

```diff
@@ -74,6 +75,7 @@ eMMC
 EmonPi
 exFAT
 FHS
+fileserver
```

Comment:
On the main website I used `file server`, which is at least supported by Wikipedia: https://en.wikipedia.org/wiki/File_server
But single word would be okay for me as well, as you prefer.

Reply:
Ok, changed to file server.